elastic/kibana
# [TSVB] Fix series containing colon

Fixes https://github.com/elastic/kibana/issues/121541

By changing the internally used separator symbol to something that's not as commonly used. I opted for the one used for the "other bucket" feature (see https://github.com/elastic/kibana/pull/81981)

To test:
* Make sure a "group by" on a field containing colons works without issues
* Check whether I forgot to change a place
